02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,57 @@
+
+let {people} = require('../data');
+
+const getPeople = (req, res) => {
+    const newPeople = people.map((person) => {
+        const { id, name } = person
+        return { id, name }
+    })
+
+    res.status(201).json( { success: true, data: newPeople } )
+}
+
+const addPerson = (req, res) => {
+    const { name } = req.body;
+
+    if (!name) {
+        return res.status(401).json( { success: false, msg: "please provide a name!" })
```

Comment:
The 401 status code is for unauthorized access.  You should use 400 here instead for Bad Request.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses